### PR TITLE
Fix capitalization of MUST NOT/SHOULD NOT

### PR DESCRIPTION
**Pull request by yurishkuro**
Per https://tools.ietf.org/html/rfc2119, the work "NOT" must also be capitalized.

Skipping changelog since this is a stylistic change.

### CODE REVIEW

File: specification/trace/api.md [hunk 1]
```diff
@@ -241,7 +241,7 @@ All mutating operations MUST return a new `TraceState` with the modifications ap
 `TraceState` MUST at all times be valid according to rules specified in [W3C Trace Context specification](https://www.w3.org/TR/trace-context/#tracestate-header-field-values).
 Every mutating operations MUST validate input parameters.
 If invalid value is passed the operation MUST NOT return `TraceState` containing invalid data
-and MUST follow the [general error handling guidelines](../error-handling.md) (e.g. it usually must not return null or throw an exception).
+and MUST follow the [general error handling guidelines](../error-handling.md) (e.g. it usually should not return null or throw an exception).
```
Comment (yurishkuro) [inline specification/trace/api.md:244]: "usually must" does not make sense

Comment (reyang) [reply specification/trace/api.md:244]: I think "and MUST follow the general error handling guidelines" is good enough - perhaps we just remove "(e.g. it usually should not return null or throw an exception)" at all?

**Review by reyang — APPROVED** [review]
LGTM with one non-blocking suggestion.

File: specification/trace/api.md (hunk 1 above)
Comment (yurishkuro) [reply specification/trace/api.md]: +1

**Review by bogdandrutu — APPROVED** [review]

File: specification/trace/api.md [hunk 2]
```diff
@@ -241,7 +241,7 @@ All mutating operations MUST return a new `TraceState` with the modifications ap
 `TraceState` MUST at all times be valid according to rules specified in [W3C Trace Context specification](https://www.w3.org/TR/trace-context/#tracestate-header-field-values).
 Every mutating operations MUST validate input parameters.
 If invalid value is passed the operation MUST NOT return `TraceState` containing invalid data
-and MUST follow the [general error handling guidelines](../error-handling.md) (e.g. it usually must not return null or throw an exception).
```
Comment (Oberon00) [inline specification/trace/api.md:244]: Why was this removed? The point about `null` is IMHO important. Maybe it should be refactored to not be inside the `e.g.`
